File: Basic/DependencyInjection/src/Transcoder/JobData.php
<?php

namespace Basic\DI\Transcoder;

class JobData
{
    private $source;
    private $target;

    public function __construct(string $source, Output $target)
    {
        $this->source = $source;
        $this->target = $target;
    }

    public function getSource(): string
    {
        return $this->source;
    }

    public function getTarget(): Output
    {
        return $this->target;
    }
}

File: oop-pattern/ch06/example/app/src/main/java/study/JobData.java
package study;

import java.io.Serializable;

public class JobData implements Serializable {

  private String source;
  private String target;

  public JobData(String source, String target) {
    this.source = source;
    this.target = target;
  }

  public JobData() {
  }

  public String getSource() {
    return source;
  }

  public String getTarget() {
    return target;
  }
}
